domain/test/db.async.test.js

```diff
@@ -128,6 +128,26 @@ const coll = 'items';
             assert(returned2[0].type === 'axe');
         });
 
+        it('should be fully to update item', async function () {
```

Comment:
"should be fully" ? What does it even mean? :) "should be success" suits better.
And even better -
"should update item"
"should not update non-existing item"